src/memmachine/episodic_memory/episodic_memory.py

```diff
@@ -345,7 +398,12 @@ async def formalize_query_with_context(
         Returns:
             A new query string enriched with context.
         """
-        episodes, summary = await self.query_memory(query, limit)
+        short_memoy, long_memory, summary = await self.query_memory(
```

Comment:
```suggestion
        short_memory, long_memory, summary = await self.query_memory(
```
I think it would also make sense to eventually swap around the order for short_memory and long_memory, because currently each list is sorted from earliest to latest. The summary is also associated with short_memory, so it would be better to have those two next to each other.
i.e. query_memory is updated to return long_memory, short_memory, summary